pipelines/resources/deploy-ztp-spokes.yaml

```diff
@@ -94,23 +94,23 @@ spec:
         workspace: ztp
 
   # Deploy Workers
-  - name: deploy-workers
-    taskRef:
-      name: spoke-deploy-workers
-    params:
-      - name: spokes-config
-        value: $(params.spokes-config)
-      - name: kubeconfig
-        value: $(params.kubeconfig)
-      - name: ztp-container-image
-        value: $(params.ztp-container-image)
-      - name: mock
-        value: $(params.mock)
-    runAfter:
-      - deploy-metallb
-    workspaces:
-      - name: ztp
-        workspace: ztp
+  # - name: deploy-workers
```

Comment:
I don't think we should maintain this as commented....in BareMetal environment is working fine.